starter_code/src/movies.js

```diff
@@ -1,20 +1,61 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
-
+function turnHoursToMinutes (duration) {
+    var duration2=duration.split(" ");
+    var durationTotal=0;
+    
+    if (duration2.length===1) {
+    durationTotal+=parseInt(duration2[0]);
+    return durationTotal;
+    } else {
+    var durationHour=parseInt(duration2[0]);
+    var durationMin=parseInt(duration2[1]);
+    var durationTotal=(durationHour*60)+durationMin;
+    }
+    return durationTotal;
+  }
 
```

Comment:
The idea here is to be able to return the whole array of movies with the duration transformed in minutes for all of them. To do so, you need to be able to access the hours and the minutes without "h" and "min" and do the mathematical operations on all the cases (movie with h and min, movie with just h and movie with just min).